nilearn/tests/test_signal.py

```diff
@@ -469,347 +483,544 @@ def test_clean_kwargs():
+        confounds=confounds1,
+        detrend=False,
+        standardize="zscore_sample",
+    )
+
+    assert_almost_equal(cleaned_signals1, cleaned_signals)
 
     # Test detrending. No trend should exist in the output.
     # Use confounds with a trend.
     temp = confounds.T
     temp += np.arange(confounds.shape[0])
 
-    cleaned_signals = nisignal.clean(signals + noises, confounds=confounds,
-                                     detrend=False, standardize=False)
-    coeffs = np.polyfit(np.arange(cleaned_signals.shape[0]),
-                        cleaned_signals, 1)
-    assert (abs(coeffs) > 1e-3).any()   # trends remain
+    cleaned_signals = clean(
+        signals + noises, confounds=confounds, detrend=False, standardize=False
+    )
 
-    cleaned_signals = nisignal.clean(signals + noises, confounds=confounds,
-                                     detrend=True, standardize=False)
-    coeffs = np.polyfit(np.arange(cleaned_signals.shape[0]),
-                        cleaned_signals, 1)
-    assert (abs(coeffs) < 1000. * eps).all()  # trend removed
+    coeffs = np.polyfit(
+        np.arange(cleaned_signals.shape[0]), cleaned_signals, 1
+    )
+    assert (abs(coeffs) > 1e-3).any()  # trends remain
+
+    cleaned_signals = clean(
+        signals + noises, confounds=confounds, detrend=True, standardize=False
+    )
+
+    coeffs = np.polyfit(
+        np.arange(cleaned_signals.shape[0]), cleaned_signals, 1
+    )
+    assert (abs(coeffs) < 1000.0 * eps).all()  # trend removed
 
     # Test no-op
     input_signals = 10 * signals
-    cleaned_signals = nisignal.clean(input_signals, detrend=False,
-                                     standardize=False)
-    np.testing.assert_almost_equal(cleaned_signals, input_signals)
-
-    cleaned_signals = nisignal.clean(input_signals, detrend=False,
-                                     standardize=True)
-    np.testing.assert_almost_equal(cleaned_signals.var(axis=0),
-                                   np.ones(cleaned_signals.shape[1]))
-
-    # Test with confounds read from a file. Smoke test only (result has
-    # no meaning).
-    current_dir = os.path.split(__file__)[0]
-
-    signals, _, confounds = generate_signals(n_features=41,
-                                             n_confounds=3, length=20)
-    filename1 = os.path.join(current_dir, "data", "spm_confounds.txt")
-    filename2 = os.path.join(current_dir, "data",
-                             "confounds_with_header.csv")
-
-    nisignal.clean(signals, detrend=False, standardize=False,
-                   confounds=filename1)
-    nisignal.clean(signals, detrend=False, standardize=False,
-                   confounds=filename2)
-    nisignal.clean(signals, detrend=False, standardize=False,
-                   confounds=confounds[:, 1])
 
+    cleaned_signals = clean(
+        input_signals,
+        detrend=False,
+        standardize=False,
+        standardize_confounds="zscore_sample",  # avoid wrong strategy warnings
```

Comment:
Is this comment referring to the deprecation warning? If so I would remove it because it's not informative and could be misleading